src/serve/index.ts

```diff
@@ -56,6 +56,8 @@ export const serve = <
    * @returns A promise that resolves to a response.
    */
   const handler = async (request: TRequest) => {
+    await debug?.log("INFO", "ENDPOINT_START");
```

Comment:
Is this redundant?

Reply:
it indicates that the endpoint was called and is executing